src/components/marchant/marchant.vue

```diff
@@ -280,7 +280,18 @@
                     <h1 class="section_title color-primary">Browse our Service Directory</h1>
                     <p class="subtitle">Select a service to see a list of verified vendors offering such service near you. </p>
                 </header>
-                <Browse />
+                <div class="card_list">
+                    <article class="card_item" v-for="browse in browses" :key="browse.id">
```

Comment:
Can you rename _browse_ and _browses_ to a more relative term where applicable?

Reply:
Resolved.